### PR TITLE
Add monitoring section to user guides

### PR DESCRIPTION
**Pull request by nrichers**
## Internal Notes for Reviewers

This PR adds initial user guide information for ongoing monitoring, including:

- New "Monitoring" section on the "Guides" landing page
- New "Ongoing monitoring" core docs
- New "Ongoing monitoring" tab under test descriptions
- Pull in new notebook for ongoing monitoring
- New glossary section for monitoring
- Other docs where monitoring should be mentioned, e.g. under "Install and initialize the developer framework" 

### New "Monitoring" section on the "Guides" landing page

image

image

**New "Ongoing monitoring" core docs**

image

### New "Ongoing monitoring" tab under "Test descriptions"

image

### Pull in new notebook for ongoing monitoring 

image

### New glossary section for monitoring

image

image

### Other docs where monitoring should be mentioned

image

## External Release Notes

Monitoring is a critical component of model risk management, as emphasized in regulations such as SR 11-7, SS1/23, and E-24. With this release of ValidMind, we officially support ongoing monitoring that you can enable for both existing and new models. 

image

Scenarios where ongoing monitoring is warranted:

- Pre-approval monitoring of new models
- Monitoring during significant updates
- Post-production monitoring

As your code runs, the monitoring template for your model automatically populates with data, providing a comprehensive view of your model’s performance over time. You access and examine these results within the ValidMind Platform UI, allowing you to identify any deviations from expected performance and take corrective actions as needed.

[Learn more ...](../../guide/monitoring/ongoing-monitoring.qmd)

### CODE REVIEW

**Review by validbeck — CHANGES_REQUESTED** [review]
Nice! I particularly love the descriptions as to what monitoring is and what is important. 

Some thoughts:

### Monitoring scenarios
- Shall we just make it consistent with our love of m-dashes here instead of colons? (Esp since the rest of the page has them already!)
- I would maybe break the paragraphs up a little as that much dense (both visually and information wise) text is hard to parse in a large chunk.

(Pushed a commit for these suggestions, as well as some other minor wording/formatting tweaks.)

### Enable monitoring for models & review monitoring results
Since this main page is already so text-heavy / sets up the context, I think it serves better as a landing page... by the time we get to the bottom it's very long.

I propose two new articles under `Ongoing monitoring` (you can then also put them in "What's next":

- Enable monitoring for models
- Review (model) monitoring results

**Comment by juanmleng** [comment]
Looks great :) A couple of minor suggestions: 

- Use Ongoing Monitoring in the glossary instead of Monitoring
- To aligned ourselves a bit with SR11-7, replace effectiveness by robustness 

For example in here: 

Screenshot 2024-08-14 at 21 38 06

I suggest: 

Ongoing Monitoring
To regularly evaluate the ongoing accuracy, robustness, and stability of a model after it has been deployed.

Ongoing Monitoring
Monitoring of model performance is a critical component of model risk management, as emphasized in SR 11-7. This process involves regularly assessing a model’s predictive accuracy, robustness, and stability to ensure that it continues to perform as expected and remains fit for its intended purpose after deployment.

Same here: 

Screenshot 2024-08-14 at 21 43 31

I suggest the same text or similar: 
Monitoring of model performance is a critical component of model risk management, as emphasized in SR 11-7. This process involves regularly assessing a model’s predictive accuracy, robustness, and stability to ensure that it continues to perform as expected and remains fit for its intended purpose after deployment.

**Comment by nrichers** [comment]
## Updates to address review feedback

@validbeck if these updates pass muster with you, could you approve this PR, please? 

### Updated wording

Now refers to "ongoing monitoring" in the glossary, uses "robustness" instead of "effectiveness", and includes references to sample regulations: 

image

image

### Separate task topics 

Reworked overview topic with some sample output from testing and child topics included below: 

image

Separate task topic for enabling monitoring:

image

Separate task topic for reviewing monitoring results:

image

**Review by validbeck — APPROVED** [review]

**Comment by nrichers** [comment]
@validbeck I updated this PR with the new style for the "Prerequisites" section, FYI. 

image

image
